coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +238,38 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        try:
+            class_padding = cls.ClassPadding(
+                *(int(padding) for padding in tuple(
+                    docstyle_settings['class_padding'])))
+        except IndexError:
+            class_padding = cls.ClassPadding('', '')
```

Comment:
because `0, 0` will mean that official defined padding is `0, 0`. So better leave the defaults to be `'', ''`